lib/src/upgrader.dart

```diff
@@ -239,6 +250,40 @@ class Upgrader {
     return true;
   }
 
+  /// Android info is fetched by parsing the html of the app store page.
+  Future<bool?> _getAndroidStoreVersion() async {
+    final id = androidId ?? _packageInfo!.packageName;
+    final uri =
+        Uri.https('play.google.com', '/store/apps/details', {'id': '$id'});
+    final response = await http.get(uri);
+    if (response.statusCode != 200) {
+      debugPrint('Can\'t find an app in the Play Store with the id: $id');
+      return null;
+    }
+    final document = parse(response.body);
+
+    final additionalInfoElements = document.getElementsByClassName('hAyfc');
```

Comment:
Is there an API contract that guarantees that `hAyfc` will always be available?

Reply:
From my research, Google doesn't change their DOM ID's often and this has been the ID for years  now.

https://stackoverflow.com/questions/34309564/how-to-get-app-market-version-information-from-google-play-store